pwnlib/adb/adb.py

```diff
@@ -1591,7 +1591,11 @@ def uninstall(package, *arguments):
 @context.quietfunc
 def packages():
     """Returns a list of packages installed on the system"""
-    packages = process(['pm', 'list', 'packages']).recvall()
+
+    # Decodes the received bytes as UTF-8 per:
+    # https://stuff.mit.edu/afs/sipb/project/android/docs/reference/java/nio/charset/Charset.html
```

Comment:
Better link to official resources.
https://developer.android.com/reference/java/nio/charset/Charset#defaultCharset()

I didn't find a charset defintion for the adb/pm commandline tool specifically, so I think this is a ok reference @Arusekk ?

Reply:
I was running into a similar issue but this seems like a more relevant source. Thank you